spectree/plugins/falcon_plugin.py

```diff
@@ -167,27 +170,59 @@ def parse_path(self, route, path_parameter_descriptions):
 
         return f'/{"/".join(subs)}', parameters
 
-    def request_validation(self, req, query, json, headers, cookies):
+    def parse_field(self, field):
+        if isinstance(field, list):
+            return [self.parse_field(subfield) for subfield in field]
+
+        # When file name isn't ascii FieldStorage will not consider it.
+        encoded = field.disposition_options.get("filename*")
+        if encoded:
+            encoding, filename = encoded.split("''")
```

Comment:
Any example for this one? I'm not sure when will the `field.filename` contains `''`.